usvm-ts/src/main/kotlin/org/usvm/machine/expr/TsExprResolver.kt

```diff
@@ -563,7 +572,9 @@ class TsExprResolver(
 
     override fun visit(expr: EtsPtrCallExpr): UExpr<out USort>? {
         // TODO: IMPORTANT do not forget to fill sorts of arguments map
-        TODO("Not supported ${expr::class.simpleName}: $expr")
+        logger.warn { "visit(${expr::class.simpleName}) is not implemented yet" }
+        scope.assert(ctx.falseExpr)
```

Comment:
This is odd code, it should definitely either be todo or at least write in the error stream

Reply:
`warn` writes to STDERR, afaik

I believe it was done just to pass the project. We are still not ready to pass the project execution even with this modification, so I think it is easier to revert most project-related adhocs in this PR.